Makefile

```diff
@@ -3,7 +3,7 @@
 
 # You can set these variables from the command line.
 SPHINXOPTS    =
-SPHINXBUILD   = sphinx-build
+SPHINXBUILD   = sphinx-build-3
```

Comment:
How is this related to the PR?
This change breaks the build for me
``The 'sphinx-build-3' command was not found.``